e2e/cypress/integration/ui/compliance/01_ssh_scan_job.spec.ts

```diff
@@ -18,100 +18,81 @@ describe('create a manual node ssh scan job and cleanup after', () => {
     // in order to be able to wait for it later
     cy.route('POST', '/api/v0/secrets/search').as('getSecrets');
 
-    // click on node credentials in sidebar
-    cy.get('chef-sidebar-entry').contains('Node Credentials').click();
-    cy.url().should('include', '/settings/node-credentials');
-
-    // wait for data to return
-    cy.wait('@getSecrets');
-
     // click on add credential button to open create form
-    cy.contains('Add Credential').click().then(() => {
-      cy.url().should('include', '/settings/node-credentials/add');
+    cy.contains('Add Credential').click();
+    cy.url().should('include', '/settings/node-credentials/add');
 
-      // fill in name for credential, username, and key
-      cy.get('form input[formcontrolname="name"]').first().type(credName);
-      cy.get('form input[formcontrolname="username"]').first()
-        .type(Cypress.env('AUTOMATE_ACCEPTANCE_TARGET_USER'));
-      cy.get('form textarea[formcontrolname="key"]')
-        .type(decoded);
+    // fill in name for credential, username, and key
+    cy.get('form input[formcontrolname="name"]').first().type(credName);
+    cy.get('form input[formcontrolname="username"]').first()
+      .type(Cypress.env('AUTOMATE_ACCEPTANCE_TARGET_USER'));
 
-      // we save the route that will be called when we navigate to the page
-      // in order to be able to wait for it later
-      cy.route('POST', '/api/v0/secrets').as('createSecret');
+    // we save some time by setting this long string as the value rather than typing it out
+    cy.get('form textarea[formcontrolname="key"]').clear().invoke('val', decoded).trigger('input');
 
-      // save the credential
-      cy.contains('Save').click().then(() => {
+    // we save the route that will be called when we navigate to the page
+    // in order to be able to wait for it later
+    cy.route('POST', '/api/v0/secrets').as('createSecret');
 
-        cy.url().should('include', '/settings/node-credentials');
+    // save the credential
+    cy.contains('Save').click();
 
-        // wait for data to return
-        cy.wait('@createSecret');
-        cy.wait('@getSecrets');
+    cy.url().should('include', '/settings/node-credentials');
 
-        // assert table has credential
-        cy.contains(credName).should('exist');
-      });
-    });
+    // wait for data to return
+    cy.wait('@createSecret');
+    cy.wait('@getSecrets');
+
+    // assert table has credential
+    cy.contains(credName).should('exist');
   });
 
   it('can create a node and attach a credential', () => {
     // navigate to node create page:
     // click on scan jobs
     cy.get('.nav-link').contains('Compliance').click();
-    cy.url().should('include', '/compliance/reports/overview');
 
-    // click into scan jobs
-    cy.get('chef-sidebar-entry').contains('Scan Jobs').click();
+    // add force to avoid waiting on the whole page to load
+    cy.get('chef-sidebar-entry').contains('Scan Jobs').click({ force: true });
```

Comment:
Slow page loading has been the main reason this test flakes out. I decided to add `force: true` to parts of the test where we're clicking buttons to go to other pages. Opening pages takes the longest load time, so I'd rather fudge those clicks so we can get to testing the actual functionality of the page we're trying to get to.